src/screens/smart-screen/smart-screen.tsx

```diff
@@ -168,7 +189,18 @@ export class SmartScreenComponent extends React.Component<IReduxProps & IExterna
 
         if (screenData) {
             if (this.state.loadingScreenData) {
```

Comment:
it's ok for now, but is not quite scalable, maybe we should have a param of function type `getLoadingSkeleton` so we can customize the skeleton where we include smart screen component and not inside the smart screen component.